File: Tasks/Python Work/bmi_category.py
weight_in_kilograms = float(input("Enter your weight (in kilograms): "))
height_in_meters = float(input("Enter your height (in meters): "))

bmi = (weight_in_kilograms)/((height_in_meters)*(height_in_meters))

if (bmi < 18.5):
	print("Underweight")
elif (bmi >= 18.5 and bmi <= 24.9):
	print("Normal")
elif (bmi >= 25 and bmi <= 29.9):
	print("Overweight")
else: print("Obese")

File: Tasks/Python Work/Java Work/BmiCategory.java
import java.util.Scanner;
	public class BmiCategory {
		public static void main(String[] args){
		Scanner input = new Scanner(System.in);

	System.out.println("Enter your weight (in kilograms)");
	double weightInKilograms = input.nextDouble();

	System.out.println("Enter your height (in meters)");
	double heightInMeters = input.nextDouble();

double bmi = (weightInKilograms / ((heightInMeters)*(heightInMeters)));

if (bmi < 18.5){
	System.out.println("Underweight");

} else if (bmi >= 18.5 && bmi <= 24.9){
	System.out.println("Normal");

} else if (bmi >= 25 && bmi <= 29.9){
	System.out.println("Overweight");

} else {
	System.out.println("Obese");
}
}
}
